ugs-core/src/com/willwinder/universalgcodesender/uielements/components/GcodeTable.java

```diff
@@ -22,58 +18,26 @@ This file is part of Universal Gcode Sender (UGS).
  */
 package com.willwinder.universalgcodesender.uielements.components;
 
-import com.willwinder.universalgcodesender.i18n.Localization;
 import com.willwinder.universalgcodesender.types.GcodeCommand;
 import java.awt.Dimension;
 import java.awt.Rectangle;
-import java.util.logging.Level;
-import java.util.logging.Logger;
 import javax.swing.JTable;
-import javax.swing.table.DefaultTableModel;
 import javax.swing.table.TableColumn;
 
 /**
+ * Customized JTable with helpers to coordinate common operations.
  *
  * @author wwinder
  */
 public class GcodeTable extends JTable {
-    private static final Logger logger = Logger.getLogger(GcodeTable.class.getName());
-
-    private DefaultTableModel model = null;
+    private GCodeTableModel model = null;
```

Comment:
Nice cleanup. In the past this widget was one of the biggest performance bottlenecks of the classic GUI, it looks like some of these changes could have helped.